pkg/blockstorage/vmware/vmware.go

```diff
@@ -251,38 +253,54 @@ var reVslmSyncFaultFatal = regexp.MustCompile("Change tracking invalid or disk i
 // SnapshotCreate is part of blockstorage.Provider
 func (p *FcdProvider) SnapshotCreate(ctx context.Context, volume blockstorage.Volume, tags map[string]string) (*blockstorage.Snapshot, error) {
 	var res types.AnyType
+	description := generateDescription(tags)
 	err := wait.PollImmediate(time.Second, defaultRetryLimit, func() (bool, error) {
-		log.Debug().Print("CreateSnapshot", field.M{"VolumeID": volume.ID})
-		task, lerr := p.Gom.CreateSnapshot(ctx, vimID(volume.ID), noDescription)
-		if lerr != nil {
-			return false, errors.Wrap(lerr, "CreateSnapshot task creation failure")
+		createErr := func() error {
+			log.Debug().Print("CreateSnapshot", field.M{"VolumeID": volume.ID})
+			task, lerr := p.Gom.CreateSnapshot(ctx, vimID(volume.ID), description)
```

Comment:
Why `lerr` and not, say, `cerr` or `cErr` or `createErrror`? It was inexplicable in the original code; no need to carry bad names forward...

Reply:
By some reason this name is used everywhere in the code, so I thought it has some meaning